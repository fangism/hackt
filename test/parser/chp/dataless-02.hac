// "dataless-02.hac"
//	$Id: dataless-02.hac,v 1.2 2007/07/31 23:24:51 fang Exp $
// void buckets

defproc void_sink(chan?() B) {
chp {
*[ B?() ]
	// error: B?() passes one null argument, 
	// user meant B? which passes no arguments
}
}

void_sink_delay X;

