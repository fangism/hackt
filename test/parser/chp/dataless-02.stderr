Fang add support for NULL lvalues!
At least one error in inst-ref-list at [7:6..7]
ERROR in one of the actions in [7:1..9]
ERROR: at least one error in CHP body.
